cmd/kured/main.go

```diff
@@ -257,7 +258,7 @@ func uncordon(nodeID string) {
 }
 
 func commandReboot(nodeID string) {
-	log.Infof("Commanding reboot")
+	log.Infof("Commanding reboot for node: %s", nodeID)
```

Comment:
This was a mistake on my part. Unfortunately I reverted #134 in #135 again. :facepalm:

Reply:
It'll be good to put it back in.